### PR TITLE
Update README with enhanced information and communication channels

### PR DESCRIPTION
**Pull request by otavio**
Enhanced the README file by adding detailed maintenance policies for the
`meta-freescale` layer, including information about branch management
and release cycles.

Additionally, included specifics about the maintained branches such as
`master`, `scarthgap`, `nanbield`, and `kirkstone`. This update aims to
provide users with a clearer understanding of the project's maintenance
practices and available branches.

Signed-off-by: Otavio Salvador <otavio@ossystems.com.br>

### CODE REVIEW

**Review by MrCry0 — APPROVED** [review]

**Review by thochstein — APPROVED** [review]
Looks good to me :-)

**Review by kraj — APPROVED** [review]
Lgtm

**Comment by angolini** [comment]
LGTM

Em qua., 13 de mar. de 2024 16:32, Khem Raj ***@***.***>
escreveu:

File: README.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+- **Latest and Greatest (Master):** Continuous attention to bug fixes and BSP upgrades, ensuring that our users have access to the latest enhancements and improvements.
+- **Stable Releases:** Each stable release is maintained until the subsequent version is released. This includes backports for BSP and critical bug fixes, along with updates for security vulnerabilities (CVEs).
+- **Long-Term Support (LTS) Releases:** These are maintained for the duration of Yocto Project's LTS support. We prioritize bug fixes, security vulnerabilities (CVEs) updates, and BSP backports to ensure stability and reliability.
+
+This policy ensures that our users can rely on consistent support and updates, backed by our commitment to delivering high-quality BSP layers.
+
+## Release Cycle
+
+- **Yocto Project LTS Releases:** Our LTS branches follow the Yocto Project LTS release schedule, which involves a four-year maintenance period. A new LTS version is typically released every two years.
```
Comment (zandrey) [inline README.md:32]: Perhaps a link to Release Schedule would be beneficial:

```suggestion
- **Yocto Project LTS Releases:** Our LTS branches follow the [Yocto Project LTS release schedule](https://wiki.yoctoproject.org/wiki/Releases), which involves a four-year maintenance period. A new LTS version is typically released every two years.
```

File: README.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+- **Stable Releases:** We aim to provide stable releases every six months, aligning with the pace of Yocto Project's stable releases.
+
+## Contributing
```
Comment (zandrey) [inline README.md:35]: I've seen that people still occasionally use [`meta-frescale` ML](https://lists.yoctoproject.org/g/meta-freescale). Shall this be mentioned here in any form (e.g., patches, discussions, etc.), or a note that ML is not used and shall be sunset?

**Review by zandrey — COMMENTED** [review]
Couple of points from my side.

File: README.md (hunk 2 above)
Comment (otavio) [reply README.md:35]: It'd be good to sunset it and adopt the **GitHub Discussions**.

**Review by zandrey — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (angolini) [reply README.md:35]: I've said in email replies to the mailing list several times that we are migrating the conversation to github.

I don't understand exactly what "sunset" means here. Is it possible to disable new emails to be sent to the mailing list?

Comment (otavio) [reply README.md:35]: @rpurdie, can we have it with an automated reply or similar?

Comment (rpurdie) [reply README.md:35]: @otavio You mean make the mailing list read only? We can do that

**Pull request merged**